packages/calling/src/CallHistory/CallHistory.ts

```diff
@@ -128,6 +143,43 @@ export class CallHistory extends Eventing<CallHistoryEventTypes> implements ICal
           );
         }
       }
+      // Check the calling backend
+      const callingBackend = getCallingBackEnd(this.webex);
+      if (callingBackend === CALLING_BACKEND.UCM) {
+        // Check if userSessions exist and the length is greater than 0
+        if (this.userSessions[USER_SESSIONS] && this.userSessions?.[USER_SESSIONS]?.length > 0) {
```

Comment:
The first if condition already checks for the existence of this.userSessions[USER_SESSIONS] right? Why do we need question marks in the `this.userSessions?.[USER_SESSIONS]?`

Reply:
Taken care of it. Removed the question marks.